contracts/ExampleNFT.cdc

```diff
@@ -133,7 +133,7 @@ access(all) contract ExampleNFT: NonFungibleToken {
                             resourceType: nil,
                             viewType: Type<MetadataViews.EVMBridgedMetadata>()
                         ) as! MetadataViews.EVMBridgedMetadata?
-                        ?? panic("Could not resolve contract-level EVMBridgedMetadata")
+                        ?? panic("Could not resolve contract-level EVMBridgedMetadata.")
```

Comment:
Imho resolving a view should never
Panic. Return nil.

Reply:
Agreed! I'll remove that. Good catch!